dom-events/buttons.js

```diff
@@ -0,0 +1,29 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log('handleClick event:', event);
+  console.log('target event:', event.target);
+}
+
+const buttonClick = document.querySelector('.click-button');
+
+buttonClick.addEventListener('click', handleClick);
```

Comment:
Don't forget the `$`! Whenever we store a DOM element to a variable, we want to make sure to include that dollar sign!